codecov.yml

```
@@ -31,7 +31,16 @@ coverage:
         # swinging coverage that is not triggered by changes in a PR.
         # See: https://twistedmatrix.com/trac/ticket/10170
         threshold: 0.02%
-
+      tests:
+        # We want a separate report just for the test code itself.
+        # If a test code is not executed it means we always skip a test
+        # or a test is accidentally always skipped.
+        # And we see that the report is green, but some bugs might be hidden
+        # due to test not being executed.
+        target: 100%
+        paths:
+          - "src/twisted/*/test/**"
+          - "src/twisted/test/**"
```

For the initial merge of this functionality it might be easier to pick a sub-set of those tests, make sure those tests have 100% coverage and add them here.

Later we can extend other tests to have 100% coverage and extend the rules

I expect that by looking for 100% covegage for the test, it should be easier to identify some code that is no longer use.

```suggestion
        paths:
          - "src/twisted/conch/test/**"
          - "src/twisted/test/**"
```